tracetransport/trace.go

```diff
@@ -36,40 +32,25 @@ func (t *tracingTransport) RegisterAuthenticatedHandler(httpMethod, path string,
 	return t.inner.RegisterAuthenticatedHandler(httpMethod, path, t.wrapHandler(httpMethod, path, h))
 }
 
+// https://github.com/GoogleCloudPlatform/google-cloud-go/blob/32a444f1bdd6d9313e6c82d90e66b599a2caa285/trace/trace.go#L171
+const (
+	httpHeader = `X-Cloud-Trace-Context`
+)
+
 func (t *tracingTransport) wrapHandler(httpMethod, path string, inner resttransport.Handler) resttransport.Handler {
 	return func(ctx context.Context, reqres resttransport.RequestResponse) error {
-		wireContext, _ := t.tracer.Extract(opentracing.HTTPHeaders, opentracing.HTTPHeadersCarrier(reqres.RequestHeader()))
-
 		spanName := t.namer.Name(httpMethod, path)
-		span := t.tracer.StartSpan(
-			spanName,
-			ext.RPCServerOption(wireContext),
-		)
+		span := t.tracer.SpanFromHeader(spanName, reqres.RequestHeader().Get(httpHeader))
```

Comment:
Can we set appropriate labels now that the pr was merged on the google trace lib?